GPU/GPUCommon.cpp

```diff
@@ -22,6 +24,12 @@ GPUCommon::GPUCommon() :
 	dumpNextFrame_(false),
 	dumpThisFrame_(false)
 {
+	DisplayList x;
+
+	// This assert failed on GCC x86 32-bit (but not MSVC 32-bit!) before adding the 
+	// "padding" field at the end. This is important for save state compatibility.
+	static_assert(sizeof(DisplayList) == 456, "Bad DisplayList size");
```

Comment:
I'm missing something.  It was 452 before only on Android x86, but something else (456?) elsewhere... and now it's 456 - everywhere?  I don't understand how adding a 32-bit field at the end aligned things.

Not complaining about this change, just want to understand it for my own benefit.

-[Unknown]